package.json

```diff
@@ -28,8 +28,8 @@
   },
   "homepage": "https://github.com/ionic-team/ionic-pwa-elements",
   "devDependencies": {
```

Comment:
As part of the work in FW-4065 it was noted that we need to update the `main` and `module` fields. Is this still needed?

Reply:
Addressed here: https://github.com/ionic-team/pwa-elements/pull/118/commits/1732d81d6530014039fbc7fbbebf2f8be6b515d7